ref/config/serverConfiguration.adoc

```diff
@@ -341,14 +276,9 @@ the context root:
 </server>
 ----
 
-=== Include processing
+== Include processing
 
-In addition to the default locations, additional config files can be
-brought in using the `include` element. When a server config file contains an
-`include` reference to another file, the server processes the contents of the
-referenced file as if they were included inline in place of the `include`
-element. In the following example, the server processes the contents of the
-`other.xml` file before processing the contents of the `other2.xml` file:
+In addition to the default locations, more configuration files can be brought in by using the `include` element. When a server config file contains an include reference to another file, the server processes the contents of the referenced file as if they were included inline in place of the `include` element. In the following example, the server processes the contents of the `other.xml` file before it processes the contents of the `other2.xml` file:
```

Comment:
I don't like "more configuration files" and configuration vs config is yet again made inconsistent in the docs.

Reply:
Updated